app/controllers/groups_controller.rb

```diff
@@ -3,7 +3,7 @@ class GroupsController < ApplicationController
 
   # GET /groups or /groups.json
   def index
-    @groups = Group.all
+    @groups = Group.all.where(user_id: current_user)
```

Comment:
Could you use an association method here?

```suggestion
    @groups = current_user.groups
```